string name
int64 id
pr2_moveit_ltl/transition[] transitions
bool is_initial_state
bool is_acceptance_state
int64[] valid_propositions
